### ISSUE TITLE
Do we still need to deal with ModuleAnimateHeat?

### ISSUE THREAD

**Issue by blowfishpro** [KSP-RO/SolverEngines#20]
https://github.com/KSP-RO/SolverEngines/blob/master/SolverEngines/EngineModule.cs#L61

It looks like `ModuleEnginesSolver` has a lot of special handling for `ModuleAnimateHeat`, however since KSP 1.0.5 most engines have moved away from that anyway.  Do we still need this code?

**Comment by NathanKell** [comment]
Yes, we do. The point is to control the emissive animations. Stock swapped to FXModuleAnimateThrottle to handle emissives more like we already do. :)

We probably should make it generic however (via IScalarModule) rather than driving ModuleAnimateHeat in particular.

**Comment by blowfishpro** [comment]
Hmm, I'm having trouble finding any engines still using `ModuleAnimateHeat` except for a couple in RO.  Switching to changing any `IScalarModule` could potentially have some unwanted side effects.
